cmd/monaco/purge/command.go

```diff
@@ -48,13 +50,14 @@ func GetPurgeCommand(fs afero.Fs) (purgeCmd *cobra.Command) {
 				return err
 			}
 
-			return purge(fs, manifestName, environment, specificApis)
+			return purge(fs, manifestName, environment, specificApis, !skipSettings)
 		},
 		ValidArgsFunction: completion.PurgeCompletion,
 	}
 
 	purgeCmd.Flags().StringSliceVarP(&environment, "environment", "e", make([]string, 0), "Deletes configuration only for specified envs. If not set, delete will be executed on all environments defined in manifest.")
 	purgeCmd.Flags().StringSliceVarP(&specificApis, "api", "a", make([]string, 0), "One or more specific APIs to delete from (flag can be repeated or value defined as comma-separated list)")
+	purgeCmd.Flags().BoolVar(&skipSettings, "skip-settings", false, "Settings are not deleted if this flag is set.")
```

Comment:
if we add a skip-settings, should we add skip-configs as well? (I'm not sure this skip is even needed for purge tbh)

Reply:
Removed the flag altogether